### DISCUSSION TITLE
Getting Audio output from system

### DISCUSSION THREAD

**Discussion by FreshImmuc** [rern/rAudio#1775] [General]
Hello,

i am trying to integrate rAudio into a bigger system but i am struggling with the following:

I need to get the audio output from rAudio and reroute it to a microphone. This should simulate the microphone picking up rAudio's output.
How can i do this? I tried with Pulseaudio, Alsa etc. but did not find a solution.

Thank you :)

**Comment by kan-ibal** [comment]
What is the real reason? This is the most ridiculous explanation I've heard. If you need a microphone please use real microphone an that's all.

**Reply by FreshImmuc** [reply]
What do you mean this is my exact use scenario. I have some apps that can only capture microphone audio. Why would someone lie in a github issue

**Reply by R101G** [reply]
@FreshImmuc 
Whatever those apps you have might be, they are not part of the rAudio image so, even if your question made sense, it is off-topic here.

**Reply by FreshImmuc** [reply]
Sure, i had hope someone knew how to do it though. But i have a non-off-topic question then:
How does rAudio handle audio output? Is there any info to that?

**Reply by kan-ibal** [reply]
Why wouldn't you use a cable between output and microphone input?
rAudio is a mpd based player.
https://mpd.readthedocs.io/en/latest/plugins.html#output-plugins
You can set output in mpd custom config. Settings->Player->Option->User's Configuration and set mpd's audio_output section.

**Reply by FreshImmuc** [reply]
Thank you! Do you know if rAudio outputs via mpd no matter what input type (e.g. bluetooth input, file playing etc.) is used?

**Comment by Editpo** [comment]
I think what you mean is you want to output from the Pi to a microphone INPUT on a recorder?
The microphone input of a recorder needs a very low level. 
I dont know of an audio board that gives a microphone level output.
You could do try with an audio hat, take the line level output and feed it into an attenuator to bring the levels down to match the microphone input.
OR you could take a feed out of the headphone socket through an attenuator

**Reply by FreshImmuc** [reply]
No thank you for this detailed explanation but this is not my use case.
I want to route the output from rAudio (system internally) to a microphone so other processes on the pi can pick up the output from rAudio.
